.travis.yml

```diff
@@ -1,21 +1,43 @@
 language: php
 sudo: false
 
-php:
-  - 7.0
-  - 5.6
-  - 5.5
-  - 5.4
-  - hhvm
-  - hhvm-nightly
-
 matrix:
+  include:
+    - php: 5.4
+      env: DEPS=lowest
+    - php: 5.4
+      env: DEPS=latest
+    - php: 5.5
+      env: DEPS=lowest
+    - php: 5.5
+      env: DEPS=latest
```

Comment:
doubling the number of tests looks a bit too much to me. Having a test for the two extrema plus one for each version is enough I think